addons/markers/XEH_postInit.sqf

```diff
@@ -18,3 +18,24 @@ GVAR(currentMarkerPosition) = [];
 GVAR(currentMarkerAngle) = 0;
 GVAR(currentMarkerColorConfigName) = "";
 GVAR(currentMarkerConfigName) = "";
+
+// set marker pos local on every computer (prevent markers visible for everyone)
+[QGVAR(applyMarkerPosLocal), {
+    params["_marker", "_pos"];
+    _marker setMarkerPosLocal _pos;
```

Comment:
To make this jip compatible, add the following here:
```
    if (isServer) then {
        private _index = (GETGVAR(allMapMarkers,[])) find _marker; // case-sensitive, but should be fine
        if (_index < 0) exitWith {ERROR_1("Could not find data for %1", _marker);};
        private _data = GVAR(allMapMarkersProperties) select _index;
        _data set [2, _pos];
    };
```

Reply:
Added this